tests/dataset/test_ocp.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import torch
+from activelearning.dataset.ocp import OCPDatasetHandler
+
+
+@pytest.fixture
+def ocp_dataset_handler_base():
+    ocp_checkpoint_path = "/network/scratch/a/alexandre.duval/ocp/runs/4648581/checkpoints/best_checkpoint.pt"
+    dataset_path = "/network/scratch/a/alexandre.duval/ocp/runs/4657270/deup_dataset"
```

Comment:
This will crash for most people because the files/dir will not exist. If you want to leave this (I'm not against it), I would at least surround it with a `try: ... except: ...` so that that the test does not fail because of that.